packages/react-core/src/components/AboutModal/__tests__/AboutModal.test.tsx

```diff
@@ -15,10 +15,12 @@ const props: AboutModalProps = {
 };
 
 describe('AboutModal', () => {
-  test('closes with escape', () => {
+  test('closes with escape', async () => {
+    const user = userEvent.setup();
+    
     render(<AboutModal {...props} isOpen />);
 
-    userEvent.type(screen.getByRole('dialog'), '{esc}');
+    await user.type(screen.getByRole('dialog'), '{Escape}');
```

Comment:
Are strings like `Escape` here something we should be using enums or constants for? So that we don't have to update all these values ever again? And can reuse them in the future? just an idea.